src/core/providers/qgssqlconnpool.h

```diff
@@ -0,0 +1,139 @@
+/***************************************************************************
+    qgssqlconnpool.h
+    ---------------------
+    begin                : January 2014
+    copyright            : (C) 2014 by Martin Dobias
+    email                : wonder dot sk at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSSQLCONNPOOL_H
+#define QGSSQLCONNPOOL_H
+#define SIP_NO_FILE
+
+#include "qgsconnectionpool.h"
+#include "qgslogger.h"
+
+/**
+ * \ingroup core
+ * \brief Template class for SQL based provider connection pool group.
+ * \note not available in Python bindings.
+ * \since QGIS 3.32
+ */
+template<typename T>
+class QgsSQLConnPoolGroup : public QgsConnectionPoolGroup<T *>
```

Comment:
```suggestion
class QgsSqlConnPoolGroup : public QgsConnectionPoolGroup<T *>
```

We follow QT conventions about acronyms case: https://docs.qgis.org/3.28/en/docs/developers_guide/codingstandards.html#functions which applies to function names and classes as well.

Reply:
Should I change it to QgsSql**Connection**PoolGroup to comply completely with the convention?

Reply:
Yes please, the more readable the better. And thank you for your patience.